### ISSUE TITLE
Incorrect Voting Period Countdown

### ISSUE THREAD

**Issue by mhagel** [hicommonwealth/commonwealth#4974]
## Describe the bug

[Cosmos] Incorrect time left on ProposalCard and in View_Proposal
- Affects only devnet sandbox communities

## To Reproduce

- https://commonwealth.im/csdk-beta/proposals (has a 90-second voting period)
- create a proposal
- When proposal displays, incorrect amount of time shown in "time left"
- Example, shows 36m when it is probably supposed to be 36s
<img width="743" alt="Screenshot 2023-09-06 at 1 07 55 PM" src="https://github.com/hicommonwealth/commonwealth/assets/9438198/b908ae94-8478-40f1-9baa-747390d140ed">

<img width="719" alt="Screenshot 2023-09-06 at 11 22 52 AM" src="https://github.com/hicommonwealth/commonwealth/assets/9438198/43adf4ab-e6f2-4ceb-907e-a3712ae3cd34">

## Expected behavior

Accurate time left on ProposalCard and in View_Proposal

**Comment by mhagel** [comment]
This only seems to occur in our sandbox devnet communities.

I have compared kyve and osmosis to other proposal sites (keplr dashboard and mintscan), and their dates agree with ours.

I'm not sure why our devnets are returning incorrect timestamps. Looking at the live logs, I can see that they are running with the correct UTC time. So this is a bit mysterious. But I have been able to confirm that our code is calculating correctly based on these timestamps.

It would be nice to figure this out for testing purposes, but we may be able to de-prioritize this for now.
